### ISSUE TITLE
[Feature request] Increase tap target y coordinate area for player progressbar, like in YT app

### ISSUE THREAD

**Issue by teaalltr** [futo-org/grayjay-android#614]
Would help usability, as one would not need to precisely tap on the exact position of the progress circle marker

**Comment by CroatZ** [comment]
Yes, exactly this please. It drives me crazy trying to tap through progress bar. Try to imitate Youtube's progress bar, they have a very good functionality of it.

**Comment by gsture** [comment]
+1 from me. Especially in the car, when using the phone in a holder, it is almost impossible to tap on the progress bar on the first try. It’s really frustrating.
